homeassistant/helpers/entity_platform.py

```diff
@@ -146,7 +146,8 @@ def async_create_setup_task():
         warn_task = hass.loop.call_later(
             SLOW_SETUP_WARNING,
             logger.warning,
-            "Setup of platform %s is taking over %s seconds.",
+            "Setup of %s platform %s is taking over %s seconds.",
+            self.domain,
```

Comment:
Should this change be included in this PR?

Reply:
Well, I could open a new PR, but I used it here, so thought I would just include it.

Reply:
Ok. It's not our regular practice, so I wanted to make sure it wasn't a change that was committed by accident.